compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -14,24 +14,38 @@ def AMDAIEBridgeToAIR : Pass<"iree-amdaie-bridge-to-air", ""> {
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEBridgeToAIRPass()";
 }
 
+def AMDAIECleanup :
+    Pass<"iree-cleanup", "func::FuncOp"> {
```

Comment:
`iree-cleanup` -> `iree-amdaie-cleanup` ?
This can be skipped because, I believe, we plan on getting rid of this pass and just keeping `canonicalize + cse` combo.

Reply:
Ok sounds good. This pass still does loop hoisting... that might be useful to keep. Ill rename the pass name anyway.